pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+
+=== Websocket and SSE entrypoints
+
+Support for Websocket (Community Edition/OSS) and Server-Sent Events (Enterprise Edition only) entrypoints are provided as separate plugins.
+
+NOTE: These plugins are expected shortly after the main 3.19 release and will be announced here when available.
+
+This feature enables you to front your event-driven backend - currently only using Kafka via the link:{{ '/apim/3.x/apim_publisherguide_introducing_kafka.html' | relative_url }}[Gravitee Community Edition / OSS Kafka connector] - with Websocket or Server-Sent Events so that API consumers can access events from the Kafka backend via one of these more consumer-friendly protocols.
+
+If an API consumer cannot connect to a native Kafka instance due to technical limitations (for example, if they cannot implement a Kafka client themselves), you can make Kafka topics available to that consumer via another, more consumer-friendly protocol such as SSE or Websocket.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-event-consumption-streaming.png %}[]
+
+This feature enables you to potentially cover use cases in the fields of Event-Driven Architecture (EDA) and Internet of Things (IoT) without the Gateway and API Management components becoming a bottleneck.
+
+NOTE: In comparison to the link:{{ '/apim/3.x/apim_event_native_api_management_overview.html#http_post_entrypoint' | relative_url }}[HTTP POST entrypoint use case] using Kafka (where the HTTP POST method is just about sending information from the client app to Kafka), with SSE and Websocket it is possible for the client app to receive information from Kafka as well.
+
+=== Webhook entrypoint
+
+This is an Enterprise Edition feature that enables you to front your backend with a link:https://en.wikipedia.org/wiki/Webhook[Webhook^].
+
+With Webhook support, an API consumer subscribes to the Webhook API, which basically indicates to the Gateway to listen for specific Kafka messages and then call the consumer to let them know that an expected event has occurred. This communication is performed via Webhook, however it is always up to the Gateway to push the events or messages to the consumer.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-event-consumption-webhook.png %}[]
+
+
+== Future planned features
+
+The GA release of the new evented Gateway providing the new policy execution engine will provide the following additional features:
+
+* Advanced Kafka connector (as an EE feature).
+* Support for more backend connector types, such as MQTT and Kinesis.
+* A dual execution mode enabling users to smoothly transition their APIs to the new execution engine, and to configure a default behavior at platform level and override it for each API level.
```

Comment:
Already implemented for jupiter (the new engine) BUT not for SME (as there is equivalent in V3)

Reply:
updated